locale/shine/extensions/mapvote/ruRU.json

```diff
@@ -0,0 +1,65 @@
+{
+	"VOTE": "Голосование за смену карты",
+	"VOTE_BOUND_MESSAGE": "{VoteDescription}. Нажмите {Button} чтобы голосовать.\nВремени осталось: %s.",
+	"VOTE_UNBOUND_MESSAGE": "{VoteDescription}.\nКарты: {MapList}.\nНаберите в консоли !vote <map>, чтобы проголосовать.\nВремени осталось: %s.\nЗадайте клавишу голосование командой \"sh_votemenu\", чтобы сделать голосование проще.",
+	"NEXT_MAP_DESCRIPTION": "Голосование за смену следующей карты началось.",
+	"NEXT_MAP_DESCRIPTION2": "Голосование за смену следующей карты в процессе.",
+	"RTV_DESCRIPTION": "Голосование за смену карты началось.",
+	"RTV_DESCRIPTION2": "Голосование за смену карты в процессе.",
+	"TIME_LEFT": "Текущая карта смениться через: %s.",
+	"TIME_LEFT_NOTIFY": "Осталось {Duration:Duration} играть на этой карте.",
+	"ROUND_LEFT_NOTIFY": "Осталось {Duration:Pluralise:is|are} {Duration} {Duration:Pluralise:round|rounds} играть на этой карте.",
```

Comment:
Need to translate `is|are` and `round|rounds`.

The idea is for it to look like:
`There is 1 round remaining on this map.` vs `There are 2 rounds remaining on this map.` etc.